src/background.js

```diff
@@ -88,14 +94,18 @@ if ('commands' in chrome) {
   };
   // bind for 60 seconds max and auto-unbind if it's a normal run
   chrome.runtime.onInstalled.addListener(onInstall);
-  setTimeout(onInstall, 60e3, {reason: 'unbindme'});
+  setTimeout(onInstall, 60e3, {
+    reason: 'unbindme'
+  });
 }
 
 // *************************************************************************
 // browser commands
 browserCommands = {
   openManage() {
-    openURL({url: '/manage.html'});
+    openURL({
+      url: 'manage.html'
```

Comment:
This robotic expansion of literals doesn't make the code more readable. On the contrary, it's less readable in such cases as this one where only one parameter is used.